modules/creating-logfilesmetricexporter.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * logging/log_collection_forwarding/cluster-logging-collector.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-logfilesmetricexporter_{context}"]
+= Creating a LogFileMetricExporter resource
+
+In {logging} version 5.8 and newer versions, the LogFileMetricExporter is no longer deployed with the collector by default. You must manually create a `LogFileMetricExporter` custom resource (CR) to generate metrics from the logs produced by running containers.
+
+If you do not create the `LogFileMetricExporter` CR, you may see a *No datapoints found* message in the {product-title} web console dashboard for *Produced Logs*.
+
+.Prerequisites
+
+* You have administrator permissions.
+* You have installed the {clo}.
+* You have installed the {oc-first}.
+
+.Procedure
+
+. Create a `LogFileMetricExporter` CR as a YAML file:
++
+.Example `LogFileMetricExporter` CR
+[source,yaml]
+----
+apiVersion: logging.openshift.io/v1alpha1
+kind: LogFileMetricExporter
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  nodeSelector: {} # <1>
+  resources: # <2>
+    limits:
+      cpu: 500m
+      memory: 256Mi
+    requests:
+      cpu: 200m
+      memory: 128Mi
+  tolerations: [] # <3>
+# ...
+----
+<1> Optional: The `nodeSelector` syntax defines which nodes the pods are scheduled on.
+<2> The `resources` syntax defines resource requirements for the `LogFileMetricExporter` CR.
+<3> Optional: The `tolerations` syntax defines the tolerations that the pods accept.
+
+. Apply the `LogFileMetricExporter` CR by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
+----
+
+.Verification
+
+A `logfilesmetricexporter` pod runs concurrently with a `collector` pod on each node.
+
+* Verify that the `logfilesmetricexporter` pods are running in the namespace where you have created the `LogFileMetricExporter` CR, by running the following command and observing the output:
++
+.Example command
```

Comment:
Question: if we're instructing users to run a command, should there at least be a generalized version of the command first, where real inputs are replaced with <> inputs? Or if this is the exact syntax that users should use in all use cases, maybe this line should be deleted as it wouldn't be an example.

Reply:
Removed line, thanks